subprojects/gradle/namespaced-resources-fixer/src/test/kotlin/com/avito/android/plugin/FileResourcesFixerTest.kt

```diff
@@ -100,80 +106,107 @@ class FileResourcesFixerTest {
     }
 
     @Test
-    fun `replace library id - simple property with alias to R`() {
+    fun `replace library id - in function argument`() {
         val app = AndroidModule("com.app.R", emptyList())
         val libraries = listOf(
-            AndroidModule("com.library", listOf("container"))
+            AndroidModule("com.library", listOf("icon"))
         )
         val result = fix(
             """
-            import com.app.R as app_R
+            import androidx.test.espresso.matcher.ViewMatchers
+            import com.app.R
+            import com.library.ViewElement
 
-            val id = app_R.id.container
+            val icon = element<ViewElement>(ViewMatchers.withId(R.id.icon))
         """.trimIndent(),
             app, libraries
         )
         assertThat(result).isEqualTo(
             """
-            import com.app.R as app_R
+            import androidx.test.espresso.matcher.ViewMatchers
+            import com.app.R
+            import com.library.ViewElement
             import com.library.R as library_R
 
-            val id = library_R.id.container
+            val icon = element<ViewElement>(ViewMatchers.withId(library_R.id.icon))
         """.trimIndent()
         )
     }
 
     @Test
-    fun `replace library id - two simple properties with the same name`() {
+    fun `replace library id - merged R has alias`() {
         val app = AndroidModule("com.app.R", emptyList())
         val libraries = listOf(
             AndroidModule("com.library", listOf("container"))
         )
         val result = fix(
             """
-            import com.app.R as app_R
-            import com.library.R as library_R
+            import com.app.R as custom_alias_R
 
-            val id1 = app_R.id.container
-            val id2 = library_R.id.container
+            val id = custom_alias_R.id.container
         """.trimIndent(),
             app, libraries
         )
         assertThat(result).isEqualTo(
             """
-            import com.app.R as app_R
+            import com.app.R as custom_alias_R
             import com.library.R as library_R
 
-            val id1 = library_R.id.container
-            val id2 = library_R.id.container
+            val id = library_R.id.container
         """.trimIndent()
         )
     }
 
     @Test
-    fun `replace library id - function argument`() {
+    fun `replace library id - library R has alias`() {
         val app = AndroidModule("com.app.R", emptyList())
         val libraries = listOf(
-            AndroidModule("com.library", listOf("icon"))
+            AndroidModule("com.library", listOf("container"))
         )
         val result = fix(
             """
-            import androidx.test.espresso.matcher.ViewMatchers
             import com.app.R
-            import com.library.ViewElement
+            import com.library.R as custom_alias_R
```

Comment:
A new case. 
The plugin added duplicated import with default alias that caused compilation errors.